features/google/src/model/importers/utils/lang-constants.js

```diff
@@ -0,0 +1,32 @@
+export const langConstants = {
+    AccessLogImporter: {
+        english: { regex: /\/Access Log Activity\/.*\.csv$/ },
+        german: { regex: /\/Zugriffsprotokollaktivitäten\/.*\.csv$/ },
+        danish: { regex: /\/Aktiviteter i adgangsloggen\/.*\.csv$/ },
+    },
+    ActivitiesImporter: {
+        english: { regex: /\/My Activity\/.*\.html$/ },
+        german: { regex: /\/Meine Aktivitäten\/.*\.html$/ },
+        danish: { regex: /\/Mine aktiviteter\/.*\.html$/ },
+    },
```

Comment:
Not important, but this could just be something like:

```
ActivitiesImporter: /\/(My Activity|Meine Aktivitäten|Mine aktiviteter)\/.*\.html$/
```